x-pack/plugins/security_solution/cypress/tasks/create_new_rule.ts

```diff
@@ -249,6 +258,31 @@ export const fillAboutRuleWithOverrideAndContinue = (rule: OverrideRule) => {
   getAboutContinueButton().should('exist').click({ force: true });
 };
 
+export const fillDefineCustomRuleWithImportedQueryAndDataViewAndContinue = (
+  rule: CustomRule | OverrideRule
+) => {
+  cy.get('[data-test-subj="rule-index-toggle-dataView"]').click();
+
+  cy.get(
+    '[data-test-subj="detectionsDataViewSelectorDropdown"] [data-test-subj="comboBoxSearchInput"]'
+  )
+    .click()
+    .type('auditbeat-*');
+  cy.get(
+    '[data-test-subj="comboBoxOptionsList detectionsDataViewSelectorDropdown-optionsList"]'
+  ).select('auditbeat-*');
+  // .click();
+
+  // cy.get('#auditbeat-*').click();
```

Comment:
Can we remove the commented out code here?

Reply:
I just got rid of the cypress stuff I added since it wasn't working anyways. I'll try to add cypress tests in a different PR.